runtime/src/main/scala/akka/grpc/javadsl/ServiceHandler.scala

```diff
@@ -10,7 +10,8 @@ import akka.annotation.ApiMayChange
 import akka.annotation.InternalApi
 import akka.grpc.scaladsl.{ ServiceHandler => sServiceHandler }
 import akka.http.javadsl.model.{ HttpRequest, HttpResponse, StatusCodes }
-// using japi because bindAndHandleAsync expects that
+
+// using japi because bind expects that.
```

Comment:
Bind should take a `akka.japi.function.Function`, right?

Reply:
Indeed. I am(a bit) lost as there are few places, I can not simply use the `akka.japi.function.Function` and `akka.japi.Function` simultaneously. Besides, I do not think, the implicit conversions could help for the java code. 
One way could be to create an additional class and do the changes there. Just like the [PatternsCS](https://doc.akka.io/japi/akka/current/akka/pattern/PatternsCS.html)  were used. 
In case, this solution is fine with you, I can do the changes.